### DISCUSSION TITLE
dependency hell

### DISCUSSION THREAD

**Discussion by DjinnDjab** [lwindolf/liferea#1266] [General]
am i doing something wrong here? im trying to install this from source on a new linux mint install and get hit with 

No package 'libxslt' found
No package 'webkit2gtk-4.0' found
No package 'json-glib-1.0' found
No package 'gobject-introspection-1.0' found
No package 'gsettings-desktop-schemas' found
No package 'libpeas-1.0' found
No package 'libpeas-gtk-1.0' found

ive already corrected a bunch so far but dang??

**Comment by DjinnDjab** [comment]
attempted compiling from git, same thing
No package 'libxslt' found
No package 'libsoup-3.0' found
No package 'webkit2gtk-4.1' found
No package 'json-glib-1.0' found
No package 'gobject-introspection-1.0' found
No package 'gsettings-desktop-schemas' found
No package 'libpeas-1.0' found
No package 'libpeas-gtk-1.0' found

gave up

**Comment by DjinnDjab** [comment]
sudo apt install   libxml2-dev libxslt1-dev libsqlite3-dev libwebkit2gtk-4.0-dev libjson-glib-dev libgirepository1.0-dev
   libpeas-dev libfribidi-dev gsettings-desktop-schemas-dev python3 libtool intltool

**Comment by DjinnDjab** [comment]
No package 'gsettings-desktop-schemas' found
No package 'libpeas-1.0' found
No package 'libpeas-gtk-1.0' found

just kidding, still errors but now i really give up

**Reply by lwindolf** [reply]
Sorry about your pain. This is really hard. I'm developing on Ubuntu so I can only provide the install instructions for most recent Ubuntu. Don't have the time to setup and try packages on other distros.

**Reply by sjehuda** [reply]
Liferea is a well known software. I suppose people of all distros would be willing to provide instructions.

Perhaps adding writing text under each distro saying "No instructions provided. Please send us a request at the issue tracker or provide us with instructions and we will publish them here".
